src/core/sessions/session.service.ts

```
@@ -122,21 +278,55 @@ export class SessionService {
                 SessionService.name,
             );
-            throw new ForbiddenAccessException();
+            throw new SessionForbiddenException();
         }
 
-        const updatedSession: SessionEntity =
-            await this.sessionRepository.updateSession(
-                sessionId,
-                updateSessionRequest,
+        try {
+            const updatedSession = await this.prisma.$transaction(async (tx) => {
```

ESLint

[error] Insert `⏎················`

(prettier/prettier)